projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/my-company/models/my-company-row.config.ts

```diff
@@ -0,0 +1,58 @@
+export enum INPUT_TYPE {
+  TEXT = 'text',
+  DATE_TIME = 'datetime',
+  NG_SELECT = 'ngSelect',
+}
```

Comment:
We have dedicated folder for models, can we move it to separate folder with something *.model.ts specific? :) Will be a good foundation for other possible models